src/action_types/admin.ts

```diff
@@ -184,4 +184,12 @@ export default keyMirror({
     REMOVED_PLUGIN: null,
     ENABLED_PLUGIN: null,
     DISABLED_PLUGIN: null,
+
+    SAML_GET_METADATA_REQUEST: null,
+    SAML_GET_METADATA_SUCCESS: null,
+    SAML_GET_METADATA_FAILURE: null,
+
+    SET_SAML_IDP_REQUEST: null,
+    SET_SAML_IDP_SUCCESS: null,
+    SET_SAML_IDP_FAILURE: null,
```

Comment:
@catalintomai I know I have not been assigned as a reviewer, but wanted to ask if you need to keep track of the requests? If not perhaps best to remove this

Reply:
@catalintomai To elaborate on this, the convention used to be to emit an action for every request, success, and failure. However, it seems that now [unused actions are being removed](https://github.com/mattermost/mattermost-redux/pull/992/files) from the repository.

Reply:
Thanks, the spurious requests should be cleaned up now.